content/intermediate/templates/resource-dependencies/index.md

```diff
@@ -179,14 +198,34 @@ There are four resources in the template snippet you pasted into your template:
 
 Let’s create a stack, and verify the above behavior. Use the AWS CloudFormation Console to [create a stack](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-console-create-stack.html) using the `resource-dependency-with-intrinsic-functions.yaml` template:
 
+:::::tabs{variant="container"}
+
+::::tab{id="cloud9" label="Cloud9"}
+1. In the **Cloud9 terminal** navigate to `code/workspace/resource-dependencies`:
+:::code{language=shell showLineNumbers=false showCopyAction=true}
+cd cfn101-workshop/code/workspace/resource-dependencies
+:::
+1. Use the AWS CLI to create the stack. The required parameter `--template-body` have been pre-filled for you.
+:::code{language=shell showLineNumbers=false showCopyAction=true}
+aws cloudformation create-stack --stack-name cfn-workshop-resource-dependencies-ref-getatt --template-body file://resource-dependencies-with-intrinsic-functions.yaml
+:::
+1. If the `create-stack` command was successfully sent, CloudFormation will return `StackId`.
```

Comment:
I got an error `An error occurred (ValidationError) when calling the CreateStack operation: Parameters: [EmailAddress] must have values`
Suggest passing in parameter to the create-stack command `--parameters ParameterKey="EmailAddress",ParameterValue="your-email-address-here`